AGOL_updater_LTCF_Data.py

```diff
@@ -95,7 +95,7 @@ def geocode(row):
 # Updated LTCF_Data is downloaded as CSV from Google Sheet 'https://docs.google.com/spreadsheets/d/1kzowz5CnFqTqzlbuZDec6JgFvLitG20q9C4iKiWpluU/edit#gid=0'
 # CSV file with updates should be named 'COVID_LTCF_Data_latest.csv'
 # Update this 'work_dir' variable with the folder you store the updated CSV in
-work_dir = r'C:\COVID19'
+work_dir = r'C:\Incidents\2020\Health_and_Medical\COVID-19_Response\Tools\COVID19-master'
```

Comment:
This isn't a huge deal, but let's just keep this as the generic 'C:\COVID19' directory, so that's what's maintained in GitHub.    Either way, one of us will have to have a different path to our COVID directory in code that we run locally.